src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/AlterTableOperation.java

```diff
@@ -0,0 +1,13 @@
+package io.stargate.sgv2.jsonapi.api.model.command.impl;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.WRAPPER_OBJECT)
+@JsonSubTypes({
+  @JsonSubTypes.Type(value = AlterTableOperationImpl.AddColumns.class),
+  @JsonSubTypes.Type(value = AlterTableOperationImpl.DropColumns.class),
+  @JsonSubTypes.Type(value = AlterTableOperationImpl.AddVectorize.class),
+  @JsonSubTypes.Type(value = AlterTableOperationImpl.DropVectorize.class)
+})
+public interface AlterTableOperation {}
```

Comment:
Other places interface and implementation are kept separate (like ColumnType, Command, etc). So maintained the same code structure.

Reply:
I am not sure that is a good reason to create separation without actual reason other than other things (for which this may make sense) do that.
Especially since this "tag" interface with no methods.

Reply:
Having said that, it's ok -- just something to consider going forward, adding things when needed and not if there is no foreseeable need.